pkg/kubelet/dockershim/network/kubenet/kubenet.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package kubenet
 
+// Defining constants for package
```

Comment:
Can you share more about what why `Golint` requires this comment? I feel the `// Defining constants for package` doesn't provide any meaningful additional info for the reader.

Would Golint also stop complaining if we instead gave each individual constant a descriptive comment?

Reply:
Golint just wants a comment for exported fields. But you're right adding one comment for the whole block doesn't really help very much. I'll update with more descriptive comments for the constants.